lib/assessment.md

```diff
@@ -22,14 +22,23 @@ Using this format will make it easier for us to grade you, so that we can get ev
 
 When we say that the web is a service, what does that mean? Explain the interaction in terms of clients and servers.
 <!-- your answer starts here -->
-
+The web is a service that uses the internet. A service has clients and servers. Someone who asks and someone who responds.
```

Comment:
This explanation is ok - I'll give you credit for it - but it could be more clear.